inc/question.class.php

```diff
@@ -402,6 +402,10 @@ private function checkBeforeSave($input) : array {
     * @return array the modified $input array
     */
    public function prepareInputForAdd($input) {
+      if (isset($input['fieldtype']) && isset($input['default_values']) && !in_array($input['fieldtype'], ['checkboxes', 'multiselect'])) {
```

Comment:
Does the initial issue only concern textarea? So it should only be applied in this case and no others.

Something like this (not tested):
```suggestion
      if (
         isset($input['fieldtype'])
         && $input['fieldtype'] == 'textarea'
         && !empty($input['default_values'])
      ) {
```

`empty()` avoids trying to `str_replace` if the key doesn't exist or its value is empty